packages/design-system/src/components/web-components/storybook.tsx

```diff
@@ -1,9 +1,12 @@
 import { Fragment } from 'react';
 
 export function webComponentDecorator(Story, context) {
+  // The Preact element `children` can have circular references which trip up Storybook's JSON evaluation
+  const { children, ...simpleArgs } = context.args;
   return (
-    <Fragment key={JSON.stringify({ ...context.args, ...context.globals })}>
-      <Story />
+    <Fragment key={JSON.stringify({ ...simpleArgs, ...context.globals })}>
+      {' '}
+      <Story />{' '}
```

Comment:
There are extra spaces here for some reason

Reply:
oh that must've carried over from copying from GH. removing!